tests/ltp/Makefile

```diff
@@ -51,19 +52,28 @@ run-sw: $(ROOT_FS)
 	@${LTP_TEST_SCRIPT} run-sw
 
 run-hw-single: $(ROOT_FS)
-	${SGXLKL_STARTER} --hw-debug $(ROOT_FS) $(test)
+	@rm -fr $(ROOT_FS_FRESH_COPY)
+	@cp -f $(ROOT_FS) $(ROOT_FS_FRESH_COPY)
```

Comment:
Since you've just removed the destination file, I don't understand why `-f` would be necessary.

Reply:
Removed.